pkgs/development/python-modules/pytest-rerunfailures/default.nix

```
@@ -1,23 +1,23 @@
-{ lib, buildPythonPackage, pythonOlder, fetchPypi, pytest, mock }:
+{ lib, pythonPackages, pythonOlder, fetchPypi, setuptools, pytest, pytestCheckHook, mock }:
```

```suggestion
{ lib
, buildPythonPackage
, pythonOlder
, fetchPypi
, setuptools
, packaging
, pytest
, pytestCheckHook
}:
```

We pass all dependencies explicitly, instead of using `pythonPackages`.